cost-analyzer/templates/query-service-deployment-template.yaml

```diff
@@ -136,5 +138,31 @@ spec:
               value: "/var/db"
             - name: CLOUD_PROVIDER_API_KEY
               value: "AIzaSyDXQPG_MHUEy9neR7stolq6l0ujXmjJlvk" # The GCP Pricing API key.This GCP api key is expected to be here and is limited to accessing google's billing API.'
+    {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+      {{ toYaml .Values.imagePullSecrets | indent 2 }}
+    {{- end }}
+      {{- if .Values.kubecostDeployment.queryService.priority }}
+      {{- if .Values.kubecostDeployment.queryService.priority.enabled }}
+      {{- if gt (len .Values.kubecostDeployment.queryService.priority.name) 0 }}
```

Comment:
I run into a bug when doing the following:

```bash
# Fail to render
helm template ./cost-analyzer --set kubecostDeployment.queryServiceReplicas=2 --set kubecostDeployment.queryService.priority.enabled=true

# Success
helm template ./cost-analyzer --set kubecostDeployment.queryServiceReplicas=2 --set kubecostDeployment.queryService.priority.enabled=true --set kubecostDeployment.queryService.priority.name=hello
```

You may need to adjust the following line?

```suggestion
      {{- if .Values.kubecostDeployment.queryService.priority.name }}
```